Automattic/WP-Job-Manager
# Update the WPJM logo

### Changes proposed in this Pull Request

* Updates the WPJM logo in the plugin to be the smiley

### Testing instructions

* Go to Addons -> Licenses page and observe
	* The updated logo in the active/inactive plugins
	* The updated logo in the admin notices (you might need to delete the `wp_job_manager_dismissed_notices` user meta if you have dismissed all notices and none appears)
